### ISSUE TITLE
More easily accessible checkmark button

### ISSUE THREAD

**Issue by epeay** [Anuken/Mindustry-Suggestions#3798]
### Describe the content or mechanics you are proposing.

On mobile, the checkmark button to commit a pending construction, is a little too close to other buttons. I frequently find myself pushing the wrong button, like jumping to a different construction sub-menu.

I'm not sure where best to place it. I'd recommend moving it to the left side of the line of icons it's already on, and perhaps let it extend past the left edge to give it some extra room.

If you're in the middle of construction, the next logical button to push is that checkmark (or cancel), right? If so, I think it makes sense to make the button easier to hit.

I tried UI scaling. 125% isn't much better, and 200% actually leaves the UI almost completely unusable.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Especially with if you go with letting the button extend off the left edge, it'll help new players understand that "this button is what you're looking for", and also for us mobile users, it'll help avoid mis-clicks.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by itcannotbe** [comment]
Get a phone with a bigger screen size.

**Comment by itcannotbe** [comment]
To be honest, the most logical location to proceed button is near the block options.

**Comment by epeay** [comment]
I have a pixel 6 pro, it's not a small phone ;-)

**Comment by epeay** [comment]
I appreciate the consideration, even if you don't ultimately implement it.

**Comment by itcannotbe** [comment]
Probably related

[Reddit Thread - Pixel 6 Pro Screen Edge Sensitivity Issues](https://www.reddit.com/r/GooglePixel/comments/qm18to/)

**Comment by epeay** [comment]
@itcannotbe I don't think the edge sensitivity is the problem, but I'm also not 100% sure it isn't.

I just opened #3802 and it got me thinking. What if we gave bottom right corner to a commit button, the same way you give the bottom left corner to the cancel button?

I can't think of a better button to give that real estate to, you know what I mean?

I did a quick mock-up to give you an idea of what's in my head.
![Screenshot_20220807-151038](https://user-images.githubusercontent.com/1848850/183307247-bbcfd606-87c9-4358-9508-221121eac474.png)
